Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/Call.hs

```diff
@@ -86,11 +87,23 @@ type FrontendBasedCallAPI =
            :> Get '[JSON] DCall.CallCallbackRes
        )
 
+type BackendBasedDriverCallApi =
```

Comment:
endpoint should be `driver/register/call/driver` and you should add the callback to capture the status callback too from exotel